### PR TITLE
Humble release versions 2024-07-29

### PR DESCRIPTION
**Pull request by audrow**
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/21608/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/15896/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/22326/)
* RHEL [Build Status](http://ci.ros2.org/job/ci_linux-rhel/1113/)

Build jobs:

https://ci.ros2.org/view/packaging/job/packaging_linux/3525/
https://ci.ros2.org/view/packaging/job/packaging_linux-aarch64/2873/
https://ci.ros2.org/view/packaging/job/packaging_linux-rhel/2025/
https://ci.ros2.org/view/packaging/job/packaging_windows/3326/
https://ci.ros2.org/view/packaging/job/packaging_windows_debug/2289/

### CODE REVIEW

File: ros2.repos [hunk 1]
```diff
@@ -34,7 +34,7 @@ repositories:
   eProsima/Fast-DDS:
     type: git
     url: https://github.com/eProsima/Fast-DDS.git
-    version: v2.6.7
```
Comment (audrow) [inline ros2.repos:37]: Should I bump here? I noticed the new patch version.

Comment (MiguelCompany) [reply ros2.repos:37]: I'd say we should bump to v2.6.8, to match what we have in rosdistro. Last humble merged PR there is [this one](https://github.com/ros/rosdistro/pull/40838)

Comment (clalancette) [reply ros2.repos:37]: Yes, agreed, let's make this v2.6.8

**Review by clalancette — CHANGES_REQUESTED** [review]

File: ros2.repos (hunk 1 above)
Comment (audrow) [reply ros2.repos:37]: Thanks, I set it to v2.6.8.

**Review by clalancette — APPROVED** [review]
This looks good to me with green CI.

**Pull request merged**